source/UncontrolledActivated.tsx

```diff
@@ -74,13 +96,15 @@ const UncontrolledActivated = ({
       ? setTimeout(onUnload, transitionDuration)
       : null
 
-    return () => {
-      clearTimeout(unloadTimeout)
+    return (): void => {
+      if (unloadTimeout) {
+        clearTimeout(unloadTimeout)
+      }
     }
   }, [isUnloading, onUnload, transitionDuration])
 
   // use parent element or fake one if it's not yet loaded
-  const parentEl = parentRef.current || pseudoParentEl
+  const parentEl = parentRef?.current || pseudoParentEl
```

Comment:
see comments 👆 around why I think this doesn't need `?`

Reply:
Just tried to remove, still complains https://i.postimg.cc/BQ6fj7cq/ss-parentref.png

Reply:
Only in the UncontrolledActivated though

Reply:
Nevermind, it was because of the ? in the Props.

Reply:
That's because line 27 has `parentRef?: RefObject<HTMLElement>` which should be able to be `parentRef: RefObject<HTMLElement>`, yah?

Reply:
(I'm running your branch right now and doing this stuff)